tomls/omnibus-base-mainnet-andromeda/spot/stataUSDC.toml

```diff
@@ -20,7 +20,7 @@ defaultValue = "<%= parseEther('100') %>"
 target = ["spotFactory.SpotMarketProxy"]
 fromCall.func = "owner"
 func = "createSynth"
-args = ["Synthetic Static aUSDC", "sStataUSDC", "<%= settings.owner %>"]
+args = ["Synthetic Static aUSDC", "sStataUSDC", "<%= settings.pdao %>"]
```

Comment:
As this change has not been published yet (release blocked by this issue), I have not incremented version